api/services/Domain.js

```diff
@@ -12,6 +12,7 @@ const { States } = Domain;
 
 /**
  * @typedef {object} DomainModel
+ * @typedef {object} SiteModel
```

Comment:
The `SiteModel` declaration should be in it's own comment block, I believe here it is going to mess with the definition of `DomainModel` which is supposed to have a property `state`.